### PR TITLE
Stops malf AI from cheesing gangs.

### PR DESCRIPTION
**Pull request by GrayRachnid**
Yes i'm butthurt
fixes https://github.com/yogstation13/yogstation/issues/2176

:cl:
rscadd: No more malf AIs blowing up gang dominators
/:cl:

### CODE REVIEW

**Comment by Daman453** [comment]
What they can do that? Reeee

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 1]
```diff
@@ -346,7 +348,7 @@
 
 	if (istype(M, /obj/machinery))
 		if(!M.can_be_overridden())
-			src << "Can't override this device."
+			src << "<span class='warning'> Can't override this device.<span>"
```
Comment (ShadowDeath6) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: `</span>`

Comment (GrayRachnid) [reply code/game/gamemodes/malfunction/Malf_Modules.dm]: oh ffs

**Comment by Time-Green** [comment]
I cant comment from my phone, but place a return after line 319, else it'll give the message and blow it up anyway

**Comment by Time-Green** [comment]
And an AI can still animate it, wich also destroys the dom

File: code/game/machinery/machinery.dm [hunk 2]
```diff
@@ -110,6 +110,7 @@ Class Procs:
 	var/power_channel = EQUIP
 		//EQUIP,ENVIRON or LIGHT
 	var/list/component_parts = null //list of all the parts used to build it, if made from certain kinds of frames.
+	var/no_malf_ai_cheesing = FALSE //Stops malf AI from blowing shit up
```
Comment (Super3222) [inline code/game/machinery/machinery.dm]: Better var name

File: code/game/gamemodes/gang/dominator.dm [hunk 3]
```diff
@@ -6,6 +6,7 @@
 	density = 1
 	anchored = 1
 	layer = HIGH_OBJ_LAYER
+	no_malf_ai_cheesing = TRUE
```
Comment (Super3222) [inline code/game/gamemodes/gang/dominator.dm]: Better var name

**Review by Super3222 — DISMISSED** [review]
Did you test this? You should consider to.

**Comment by ShadowDeath6** [comment]
I fixed the op for you but if it's related to an issue put fixes (issue number) in the body

**Comment by GrayRachnid** [comment]
@Super3222  it's been tested

**Comment by ShadowDeath6** [comment]
@Super3222 if you want to re-review go ahead, I dismissed yours so people don't think it's stale

**Comment by Super3222** [comment]
Just let me come back and do it next time, ok? It's important that us ("maintainers") help review these PR's, and maybe even test them, so Cruix isn't always taking care of the load. I don't want broken PRs merged just because they look stale with my requesting review on there. Contact me on Discord.

**Comment by ShadowDeath6** [comment]
don't push from master also

**Comment by GrayRachnid** [comment]
it was mistake

**Pull request merged**